server/mcp_server_domain_service/src/domain_openapi/server.py

```diff
@@ -0,0 +1,21 @@
+# coding:utf-8
+
+from src.domain_openapi.mcp_server import create_mcp_server
+from dotenv import load_dotenv
+import asyncio
+import sys
+
+load_dotenv()
+
+
+def main():
+    try:
+        mcp = create_mcp_server()
```

Comment:
pls support both sse and stdio transport, by default local deployment enable stdio

Reply:
We currently only support local mode. Would that be okay for you?

Reply:
pls try to support sse later after the first merge.